itou/openid_connect/models.py

```diff
@@ -106,8 +112,8 @@ def create_or_update_user(self, is_login=False):
         Create or update a user managed by another identity provider.
          - If there is already a user with this username (user_info_dict["sub"])
            and from the same identity provider, we update and return it.
-         - If there is already a user with the email, we return this user.
-         - otherwise, we create a new user based on the data we received.
+         - If there is already a user with the email, we return this user, provided
+           the email is not in use by another account (some providers allow email overloading)
```

Comment:
```suggestion
           the email is not in use by another account of the same provider (some providers allow email overloading),
           - otherwise, we create a new user based on the data we received.
```